scripts/CLI_VERSION

```diff
@@ -1 +1 @@
-1.15.0-next-1629322356000
+1.15.0-next-1629487941000
```

Comment:
note: this is the commit after the protocol changes, not ToT to reduce the scope of this PR.